mutiny-core/Cargo.toml

```diff
@@ -41,6 +41,12 @@ aes = { version = "0.8" }
 jwt-compact = { version = "0.8.0-beta.1", features = ["es256k"] }
 argon2 = { version = "0.5.0", features = ["password-hash", "alloc"] }
 
+fedimint-client = { git = "https://github.com/fedimint/fedimint", branch = "master" }
+fedimint-core = { git = "https://github.com/fedimint/fedimint", branch = "master" }
+fedimint-wallet-client = { git = "https://github.com/fedimint/fedimint", branch = "master"}
```

Comment:
We might want to pick a commit and stick to it until 0.2.0, otherwise we might get random compile errors anytime they change an api